src/lib/Preview.js

```diff
@@ -502,14 +502,24 @@ class Preview extends EventEmitter {
         }
     }
 
+    /**
+     * Checks if the file being previewed supports printing.
+     *
+     * @public
+     * @return {boolean}
+     */
+    canPrint() {
+        return !!canDownload(this.file, this.options) && checkFeature(this.viewer, 'print');
```

Comment:
Is the `!!` necessary here?

Reply:
Both return booleans, but figured to ensure it is a boolean in case those interfaces change, I don't believe it hurts anything.

Reply:
Seems like the `!!` should be omitted if it's not needed.